.babelrc.js

```diff
@@ -4,6 +4,15 @@ module.exports = {
         '@babel/preset-typescript',
         '@babel/preset-react',
     ],
+    plugins: [
+        'const-enum',
```

Comment:
It'd be great to add a link in a comment to https://babeljs.io/docs/en/babel-plugin-transform-typescript#caveats as I wasn't sure why this was needed :)